core/src/test/scala/matryoshka/instances/fixedpoint/stream.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2014–2016 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package matryoshka.instances.fixedpoint
+
+import matryoshka.{ToIdOps => toIdOps, _}, Recursive.ops._
+
+import scala.Int
+
+import org.specs2.ScalaCheck
+import org.specs2.mutable._
+import scalaz._, Scalaz._
+
+class StreamSpec extends Specification with ScalaCheck {
+  // fixes implicit resolution
+  type IntStream[A] = (Int, A)
+
+  /** Continues the Fibonacci sequence from a seed pair of values.
+    */
+  val fib: Coalgebra[(Int, ?), (Int, Int)] = {
+    case (p2, p1) =>
+      val c = p1 + p2
+      (c, (p1, c))
+  }
+
+  /** Generates an infinite stream of the carrier value.
+    */
+  def constantly[A]: Coalgebra[(A, ?), A] = i => (i, i)
+
+  "fib" should {
+    "begin with 1" in {
+      // TODO: implement `take`
+      (0, 1).ana[Nu, IntStream](fib).project._1 must_== 1
+    }
+
+    "lazily generate the correct sequence" in {
+      // TODO: implement `take`
+      (0, 1).ana[Nu, IntStream](fib).project._2.project._2.project._2.project._2.project._2.project._1 must_== 13
```

Comment:
I _think_ drop and take can be implemented as unfolds to Stream and List, respectively, but I haven't tried it yet.